src/client/src/OpenFin/Window/WindowViewport.tsx

```diff
@@ -1,3 +1,8 @@
+// disabling these rules for the time being as there are
```

Comment:
as earlier, previous goes at this have led me towards centralising exceptions in overrides in the eslint config, then the various flavours of errors/warnings can be managed and corrected in one commit/PR ..